src/ovp_pipeline/commands/promote.py

```
@@ -151,13 +161,24 @@ def _run_concept(args: argparse.Namespace) -> int:
             )
             actions["rejected_files"].append(str(path))
 
+    if args.apply and actions["promoted"]:
+        try:
+            knowledge_index_result = rebuild_knowledge_index(vault_dir, pack_name=pack.name)
+            knowledge_index_rebuilt = True
+        except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)